protocols/kad/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.44.7 - unreleased
```

Comment:
Same here, `0.44.6` isn't released yet so we don't need to add this here.

Reply:
fixed b87b155